src/components/Alert/DismissibleAlert.test.jsx

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import user from '@testing-library/user-event';
+import React from 'react';
+
+import DismissableAlert from './DismissibleAlert';
+
+describe('<DismissableAlert />', () => {
+	const titleText = 'A Test Component';
+
+	describe('Dismiss behavior', ()=> {
+		it('should include dismiss button', () => {
+      const customDismissBehavior = jest.fn();
```

Comment:
Is prettier on your branch already? Minor feedback but looks like the indent might be off a little here.

Reply:
It was but the files had already been committed. I manually ran prettier on the project.